corehq/apps/registration/templates/registration/email/base_templates/_base_container_normal.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'registration/email/base_templates/_base.html' %}
```

Comment:
can't you just wrap this content in a container block in `_base.html` and override it in templates that want to override it?

Reply:
see change in `bmb/marketing-styling-deploy+ad/use-project-logo-in-invite-email+fr/rich-text-emails`